orchestrator/forms/validators.py

```diff
@@ -293,29 +273,6 @@ def __modify_schema__(cls, field_schema: Dict[str, Any]) -> None:
         field_schema.update(format="label", type="string")
 
 
-class MigrationSummary(DisplayOnlyFieldType):
-    data: ClassVar[Optional[SummaryData]] = None
-
-    @classmethod
-    def __modify_schema__(cls, field_schema: Dict[str, Any]) -> None:
-        field_schema.update(format="summary", type="string", uniforms={"data": cls.data})
-
-
-def migration_summary(data: Optional[SummaryData] = None) -> Type[MigrationSummary]:
-    namespace = {"data": data}
-    return new_class("MigrationSummaryValue", (MigrationSummary,), {}, lambda ns: ns.update(namespace))
-
-
-class MTU(ConstrainedInt):
```

Comment:
I left the mtu and vlan stuff in since we seem to be network centric.. Or should we make a seperate lib with Network based extensions? This seems usefull for others..